autoload/ale/lsp_linter.vim

```diff
@@ -130,6 +130,11 @@ function! ale#lsp_linter#HandleLSPResponse(conn_id, response) abort
         call s:HandleLSPErrorMessage(l:linter_name, a:response)
     elseif l:method is# 'textDocument/publishDiagnostics'
         call s:HandleLSPDiagnostics(a:conn_id, a:response)
+    elseif l:method is# 'window/showMessage'
+        call ale#lsp#window#showMessage(
```

Comment:
I think we should probably add a setting, which might be disabled by default, for showing these messages. I imagine they could be annoying to a lot of users.

Reply:
https://github.com/dense-analysis/ale/pull/2652/files#r309254007

Reply:
Closing this (being discussed elsewhere in this PR)